README.md

```diff
@@ -140,26 +140,31 @@ This package contains some launch files, depending on the nodes you want to run.
 - **`robot_controls.launch.py`**: this launch file allows to run the `controller manager` node for ros2_control and the nodes for the single controllers (one for each part).
 - **`circle_demo.launch.py`** and **`grasp_demo.launch.py`**: as the name suggests, they are two examples of commanding the robot in the cartesian space using `torso + right_arm` as planning group.
 
-
 Before running the nodes, make sure that your environment variable `YARP_ROBOT_NAME` is properly set with the name of your robot. If not, set it for each shell according to the chosen model, for example:
 
 ```shell
-export YARP_ROBOT_NAME="icub"
+export YARP_ROBOT_NAME="iCubGenova11"
+```
+
+or, for simulated models:
+
+```shell
+export YARP_ROBOT_NAME="iCubGazeboV2_5"
 ```
 
-### icub_moveit_config
+### xcub_icub_moveit_config
 
 This package contains the configuration files to make iCub working with MoveIt2. In particular, each of the parts of iCub robot (head, left_arm, right_arm, torso, left_leg and right_leg) are defined in terms of `planning group`, and for each of them a ros2_control of type `FollowJointTrajectory` is set.
 
-### ergocub_moveit_config
+### xcub_ergocub_moveit_config
```

Comment:
Isn't better `ergocub_moveit_config` ? why we need the xcub prefix? maybe I missed something @traversaro @martinaxgloria